docs/maintainers/sass-compilation.md

```diff
@@ -0,0 +1,47 @@
+# play-frontend-hmrc: SASS compilation
+
+This library manages SASS compilation for you. However, should you wish to add your own for any reason using 
+`sbt-sassify`, please read the steps below.
+
+1.  Import the required styles in your `app/assets/stylesheets/application.scss` file:
+    ```scss
+    $govuk-assets-path: "/<route-to-your-service>/assets/lib/govuk-frontend/govuk/assets/";
+    $hmrc-assets-path: "/<route-to-your-service>/assets/lib/hmrc-frontend/hmrc/";
+    
+    @import "lib/govuk-frontend/govuk/all";
+    @import "lib/hmrc-frontend/hmrc/all";
+    ```
+Ensure [sbt-sassify](https://github.com/irundaia/sbt-sassify) is added to plugins.sbt
+
+1.  Add routes for hmrc-frontend and govuk-frontend assets in `conf/app.routes`:
+    ```scala
+    ->         /govuk-frontend                     govuk.Routes
+    ->         /hmrc-frontend                      hmrcfrontend.Routes
+    ```
+
+1. Ensure you have the correct routing for all other static assets including the compiled Javascript and images provided
+   by the hmrc-frontend webjar:
+    ```
+    GET        /assets/*file                        controllers.Assets.versioned(path = "/public", file: Asset)
+    ```
+
+1. Update your scripts template to include the hmrc-frontend javascript assets:
```

Comment:
I would recommend keeping using hmrcScripts but pass in their own scripts template.